src/Pickles/Pickles.TestFrameworks/CucumberJson/CucumberJsonSingleResultLoader.cs

```diff
@@ -31,7 +31,14 @@ public class CucumberJsonSingleResultLoader : ISingleResultLoader
     {
         public SingleTestRunBase Load(FileInfoBase fileInfo)
         {
-            return new CucumberJsonSingleResults(this.ReadResultsFile(fileInfo));
+            List<Feature> features = this.ReadResultsFile(fileInfo);
+            if (features == null)
+            {
+                Console.WriteLine("There was an issue with the results format for " + fileInfo.Name);
```

Comment:
You're writing to Console.WriteLine instead of using the Logging infrastructure.